### PR TITLE
Add versions

### PR DESCRIPTION
**Pull request by Henry-WattTime**

### CODE REVIEW

**Comment by Henry-WattTime** [comment]
@nickamuchwatt review and approve?

File: sgip_api.py [hunk 1]
```diff
@@ -20,6 +20,7 @@
 # starttime and endtime are optional, if ommited will return the latest value
 starttime = '2020-03-01T00:00:00-0000'  # UTC offset of 0 (PDT is -7, PST -8)
 endtime = '2020-03-01T00:45:00-0000'
+moerversion = '1.0' #'2.0'
```
Comment (nickamuchwatt) [inline sgip_api.py]: this thing is just called 'version' in the api, so you should just call it 'version' or if you want to be specific (which is probably good), call it moer_version

I think 'moerversion' might be the param in /data or something?

File: sgip_api.py [hunk 2]
```diff
@@ -52,25 +53,25 @@ def login(username, password):
...
 
 
-def moer(token, ba, starttime=None, endtime=None):
+def moer(token, ba, starttime=None, endtime=None, version=None):
     url = 'https://sgipsignal.com/sgipmoer'
     headers = {'Authorization': 'Bearer {}'.format(token)}
     params = {'ba': ba}
     if starttime:
-        params.update({'starttime': starttime, 'endtime': endtime})
+        params.update({'starttime': starttime, 'endtime': endtime, 'version': version})
```
Comment (nickamuchwatt) [inline sgip_api.py]: you don't want to send a version of None.  so you should remove version from this line and have another if statement
if version:
       params['version'] = version

Comment (nickamuchwatt) [reply sgip_api.py]: same for forecast

**Review by nickamuchwatt — COMMENTED** [review]
see comments, good otherwise

File: sgip_api.py (hunk 1 above)
Comment (Henry-WattTime) [reply sgip_api.py]: change to moer_version to be specific (and avoid confusion with forecast version)

File: sgip_api.py (hunk 2 above)
Comment (Henry-WattTime) [reply sgip_api.py]: Corrected!

**Review by nickamuchwatt — APPROVED** [review]
``

**Pull request merged**